androidTest/java/com/owncloud/android/authentication/AuthenticatorActivityTest.java

```diff
@@ -271,7 +271,7 @@ public void test3_check_login()
         SystemClock.sleep(WAIT_INITIAL_MS);
 
         // Check that login button is disabled
```

Comment:
You can update this comment to `Check that login button is hidden`

Reply:
Done!